frontend/components/PlatformCompatibility/PlatformCompatibility.tsx

```diff
@@ -65,7 +65,10 @@ const PlatformCompatibility = ({
   return (
     <span className={baseClass}>
       <b>
-        <TooltipWrapper tipContent="Estimated compatiblity based on <br /> the tables used in the query.">
+        <TooltipWrapper
+          tipContent="Estimated compatiblity based on <br /> the tables used in the query."
+          tipDelay
```

Comment:
For boolean variable/props we want to use names that have a true/false answer. For this, `isDelayed` would be more suitable.

Reply:
Fixed!